docs/documentation/platform/sso/overview.mdx

```diff
@@ -39,18 +39,30 @@ If your required identity provider is not shown in the list above, please reach
   For enhanced security, Infisical enforces PKCE (Proof Key for Code Exchange) with the OAuth 2.0-based SSO providers and OIDC. This provides additional protection against authorization code interception attacks and strengthens your authentication flow security.
 </Info>
 
+## Admin Login Portal
+
+Organization Admins can utilize the Admin Login Portal to bypass SSO enforcement in case of an emergency.
+
+This portal is accessible at `/login/admin` (e.g., https://app.infisical.com/login/admin).
+
+<Note>
+    This bypass functionality is exclusively available to **Organization Admins**. **Server Admins** are not permitted to use this feature.
```

Comment:
> **Server Admins** are not permitted to use this feature.

this might be confusing for people. Let's say something like, in order to bypass SSO for a organization, you must have admin role at the organization. This is independent from server admin....